python/ray/rllib/optimizers/async_replay_optimizer.py

```diff
@@ -299,6 +311,54 @@ def stats(self, debug=False):
         return stat
 
 
+@ray.remote(num_cpus=0)
+class BatchReplayActor(object):
+    """The batch replay version of the replay actor.
+
+    This allows for RNN models, but ignores prioritization params.
+    """
+
+    def __init__(self, num_shards, learning_starts, buffer_size,
```

Comment:
can you document?

Reply:
No, it's not worth it for the internal optimizers. They are documented in the agent apis.